packages/tupaia-web-server/src/routes/EmailDashboardRoute.ts

```diff
@@ -67,9 +72,9 @@ export class EmailDashboardRoute extends Route<EmailDashboardRequest> {
     )) as Pick<DashboardMailingList, 'id'>[];
 
     if (!mailingList) {
-      return {
-        message: `There is no mailing list for dashboard: ${dashboardName} at: ${entity.name} in project: ${projectEntity.name}`,
-      };
+      throw new Error(
```

Comment:
Decided this is actually more of an error case, since the email button shouldn't appear in the UI if there's not mailing list for the dashboard